nano/store/lmdb/peer.cpp

```diff
@@ -4,12 +4,25 @@
 nano::store::lmdb::peer::peer (nano::store::lmdb::component & store) :
 	store{ store } {};
```

Comment:
spurious semicolon and lesser point, the braces placement looks a bit strange stylistically